apps/studio/prisma/migrations/20240807033704_add_site_theme_column_to_site_table/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Site" ADD COLUMN     "theme" JSONB;
```

Comment:
not sure i understand this, why would theme be a jsonb here rather than just an enum? storing jsonb implies there's some config that we are taking into studio and by extension, allowing the user to customise but i thought the point of using the render engine was to have set themes, so allowing customisation on the theme level seems abit surprising to me. not sure if there's something i'm missing? (there probably is)